src/main/java/org/assertj/core/api/InstanceOfAssertFactories.java

```diff
@@ -0,0 +1,68 @@
+package org.assertj.core.api;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.function.DoublePredicate;
+import java.util.function.IntPredicate;
+import java.util.function.LongPredicate;
+import java.util.function.Predicate;
+
+/**
+ * Static {@link InstanceOfAssertFactory InstanceOfAssertFactories} for {@link Assert#asInstanceOf(InstanceOfAssertFactory)}.
+ *
+ * @since 3.13.0
+ */
+public interface InstanceOfAssertFactories {
+
+  InstanceOfAssertFactory<Predicate, PredicateAssert<Object>> PREDICATE = new InstanceOfAssertFactory<>(Predicate.class,
```

Comment:
Fix the warning with `@SuppressWarnings("rawtypes")`

Reply:
I wanted to play a bit more to see if I can avoid the suppression, if not I'll use it